index.html

```diff
@@ -33,7 +33,13 @@
         <h1>Upcoming Events</h1>
         <div>
             <div>
-                Thursday, March 7, 2024, 5:00pm — <a href="https://www.niwa.org/day1">Cherry Blossom Festival @ Japanese Friendship Garden & Museum, Balboa Park</a>
+                Saturday, August 10, 2024, 11:00am — <a href="https://www.niwa.org/odori">Toro Nagashi Festival @ Japanese Friendship Garden & Museum</a>
+            </div>
+            <div>
+                Saturday, August 10, 2024, 4:30pm — <a href="https://checkout.square.site/buy/YRKCWPGPWTGJPWITRNPJIZGS">Brew Game Plus VI @ Aztec Brewing Company</a>
+            </div>
+            <div>
+                Saturday, October 19, 2024, 1:30pm — <a href="https://www.niwa.org/otaku">Otaku Smash Fest @ Japanese Friendship Garden & Museum</a>
```

Comment:
Should probably remove this until they add us to their website.

Reply:
now "hidden"